cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -20,49 +21,66 @@ import (
 const (
 	// GasLimit the gas limit in Gwei used for transactions sent with TransactOpts
 	GasLimit = uint64(3000000)
+	// GasForMint mint a cosmos native token in Ethereum
+	GasForMint = int64(282031)
```

Comment:
Why not just make this a big.int when creating it instead of doing it like this and casting it later?

Reply:
grammer limit.